.github/workflows/test-sim.yml

```diff
@@ -36,3 +37,10 @@ jobs:
       - name: Simulation multi thread multi node test
         run: yarn test:sim:multiThread
         working-directory: packages/lodestar
+
+      - name: Upload debug log test files
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: debug-test-logs
```

Comment:
I would include commit hash had some weird issues where other runs would overwrite files with same name

Reply:
That's so weird, in which repo did that happen? Is this a known public issue in Github Actions runners?

Reply:
private one, just checked it, it appears that issue was only when downloading artifact in another job.